src/bitboxbase/bitboxbase.c

```diff
@@ -45,7 +49,16 @@ int main(void)
     hww_setup();
     common_main();
     traceln("%s", "Device initialized");
+    bitboxbase_watchdog_init();
+    bitboxbase_screensaver_init();
+    bitboxbase_background();
     for (;;) {
+        if (bitboxbase_watchdog_check() && bitboxbase_state_get() > 0) {
```

Comment:
is that the same as `bitboxbase_state_get() != BBBWaiting`? If so, I'd prefer this statement.  `>0` is a bit magical, even crossing into semantics of another language ;)

Reply:
hehe, yeah. I agree `!= BBBWaiting` is easier to read. Since C enums aren't really types I tend to use numbers with them, but it's a bad habit.

Reply:
for my understanding, this is not a C enum but a Rust enum, right? they just both happen to work the same way.